truckersmp_cli/utils.py

```diff
@@ -481,6 +482,9 @@ def wait_for_steam(use_proton, loginvdf_paths, wine=None, env=None):
                 waittime).format(waittime), end="")
             time.sleep(1)
             waittime -= 1
+            if use_proton and Args.native_steam_dir != "auto":
+                # don't compare timestamps because the path is already given
+                continue
```

Comment:
Instead of skipping everything we should add the path to our list so we can actually wait correctly.